corehq/apps/app_manager/templates/app_manager/case_summary.html

```diff
@@ -116,17 +118,78 @@ <h4 class="panel-title panel-title-nolink">
                                             <!-- /ko -->
                                         </td>
                                         <td>
-                                            <div data-bind="template: {name: 'load-save-questions', data: {questions: load_questions, getIcon: $root.questionIcon}}"></div>
+                                            <ul class="fa-ul" data-bind="foreach: load_questions"><!-- Each of these is an object containing a condition and a question -->
```

Comment:
What would you think of extracting this as a django template to be more DRY? I've also having trouble following why the load_questions and save_questions usages are now slightly different, why is that?